src/app/venter/complaints-home/complaints-home.component.css

```diff
@@ -0,0 +1,100 @@
+.container {
+    margin-right: 18%;
+    margin-left: 18%;
```

Comment:
Combine as `margin: 0 18%`

Reply:
Also, it's usually a really bad idea to specify such big margins in percent; instead make the width and height correct and specify margins in an absolute unit.